Documentation/ceph-examples.md

```diff
@@ -43,27 +41,37 @@ Settings for the operator are configured through environment variables on the op
 Now that your operator is running, let's create your Ceph storage cluster:
 - `cluster.yaml`: Common settings for a production storage cluster. Requires at least three nodes.
 - `cluster-test.yaml`: Settings for a test cluster where redundancy is not configured. Requires only a single node.
-- `cluster-minimal.yaml`: Brings up a cluster with only one mon and a mgr so the Ceph dashboard can be used for the remaining cluster configuration.
+- `cluster-minimal.yaml`: Brings up a cluster with only one [ceph-mon](http://docs.ceph.com/docs/mimic/man/8/ceph-mon/) and a [ceph-mgr](http://docs.ceph.com/docs/mimic/mgr/) so the Ceph dashboard can be used for the remaining cluster configuration.
```

Comment:
instead of mimic, let's use nautilus links

Reply:
Updated links to Nautilus